src/popup/pages/Lock/index.js

```diff
@@ -118,20 +235,34 @@ class LockPage extends React.Component {
                 </div>
                 {this.renderPwdInput()}
             </div>
+            <div className={"restore-bottom-container"} onClick={this.onClickRestore}>
+                <p className="restore-bottom">{getLanguage('resetWallet')}</p>
```

Comment:
Move `onClick` to `.restore-bottom` element (because `.restore-bottom` has `cursor: pointer`)